_posts/2020-03-27-Manage-dotfiles-with-Git.md

```diff
@@ -0,0 +1,92 @@
+---
+author: Sébastien 'sogal' Poher
+date: 2020-03-27 12:00:00+01:00
+layout: post
+title: Manage your dotfiles with Git
+image: /assets/images/2020-03-27/git_logo.png
+categories:
+- Leap
+- Tumbleweed
+- Weekly News
+tags:
+- openSUSE
+- Tumbleweed
+- git 
+- configuration
+- shell
+
+---
+
+## Dot what ???
+
+What is commonly referred to as *dotfiles* are all those small plain text files that contains your softwares' configuration. Most of the time they reside in your `$HOME` directory but are hidden as they are prefixed with a dot, hence their name. For some apps, you can find them as well in the `$HOME/.config` directory.
```

Comment:
Use plural instead of singular: […] that contain<b>s</b> your […]

Reply:
Fixed.
I also updated the README.md after LCP's remark: I add a line about where to put images used in submitted articles.